docs/index.rst

```diff
@@ -1,11 +1,36 @@
-*******************************************
-Welcome to TransferBench's documentation!
-*******************************************
+****************************
+TransferBench documentation
+****************************
 TransferBench is a simple utility capable of benchmarking simultaneous transfers between user-specified devices (CPUs/GPUs).
```

Comment:
Perhaps a little tidying up in the intro:
TransferBench is a utility capable of benchmarking simultaneous transfers between user-specified devices (CPUs/GPUs). A transfer is a single operation where an executor reads and adds values from source (SRC) memory locations, then writes the sum to destination (DST) memory locations. This simplifies to a simple copy operation when dealing with a single SRC/DST.

Reply:
Fixed